rust/README.md

```diff
@@ -307,7 +307,7 @@ To connect to the database, do the following:
 ```rust
 use scylla::{Session, SessionBuilder};
 
-fn main() {
+async fn main() {
     let uri = "127.0.0.1:9042";
     let session: Session = SessionBuilder::new().known_node(uri).build().await?;
 }
```

Comment:
Please either add `Result<(), Box<dyn Error>>` as the return type for `main`, or use `unwrap()` instead of the question mark (`?`) operator.

Reply:
Agreed. I think the example is a little lacking at the moment. An example that would actually compile might look something like:

```
use scylla::{Session, SessionBuilder, IntoTypedRows};
use std::error::Error;

#[tokio::main]
async fn main() -> Result<(), Box<dyn Error>> {
    let uri = "127.0.0.1:9042";

    let session: Session = SessionBuilder::new().known_node(uri).build().await?;

    if let Some(rows) = session.query("SELECT a, b, c FROM ks.t", &[]).await?.rows {
        for row in rows.into_typed::<(i32, i32, String)>() {
            let (a, b, c) = row?;
            println!("a, b, c: {}, {}, {}", a, b, c);
        }
    }

    Ok(())
}
```

with deps

```
[dependencies]
scylla = "0.3"
tokio = {version = "1.1.0", features = ["full"]}
```


FWIW I see the same potential issue with https://github.com/scylladb/scylla-rust-driver#examples ... Might be safer to assume best example usage should be async and make this consistent across all example code. I think this link https://rust-driver.docs.scylladb.com/stable/quickstart/example.html is more consistent for example.